example/src/SnackBar/SnackBarPage.tsx

```diff
@@ -19,12 +19,21 @@ export const SnackBarPage = () => {
                 iconName="close"
                 message={message}
             />
+            <SnackBar
+                visible={isVisible}
+                actionLabel="Longer action"
+                onClose={() => setVisible(false)}
+                callBack={() => console.log('callBack')}
+                iconName="close"
+                message={message}
+            />
```

Comment:
Would it be possible to manage snackbar props like native paper, i find it cleaner:
```
      <Snackbar
        visible={visible}
        onDismiss={onDismissSnackBar}
        action={{
          label: 'Undo',
          onPress: () => {
            // Do something
          },
        }}>
        Hey there! I'm a Snackbar.
      </Snackbar>
```

instead of having an actionLabel, and an onPress they have one action prop composed of a label and an onPress callback.